src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/CreateOrganizationOverlay.tsx

```diff
@@ -92,18 +92,37 @@ import {getOrgCreationAllowancesThunk} from 'src/identity/allowances/actions/thu
 import {generateProviderMap} from 'src/identity/components/GlobalHeader/GlobalHeaderDropdown/CreateOrganization/utils/generateProviderMap'
 import {SafeBlankLink} from 'src/utils/SafeBlankLink'
 
+// Eventing
+import {event} from 'src/cloud/utils/reporting'
+import {
+  CreateOrgOverlayEvent,
+  multiOrgTag,
+} from 'src/identity/events/multiOrgEvents'
+
 const switchToNewOrgButtonStyle = {
   color: 'black',
   textDecoration: 'underline',
   cursor: 'pointer',
 }
 
-const SwitchToNewOrgButton = (url: string): JSX.Element => {
+const SwitchToNewOrgButton = (
+  url: string,
+  newOrg: CreatedOrg,
+  currentOrgId
+): JSX.Element => {
+  const handleEventing = () => {
+    event(CreateOrgOverlayEvent.SwitchToNewOrg, multiOrgTag, {
+      currentOrgId,
+      createdOrgId: newOrg.id,
+      createdOrgName: newOrg.name,
+    })
+  }
```

Comment:
I thought about the names for these keys a bit more: I think we should mimic the names of the keys that are used for switching orgs in the other location that occurs in the app (the Org Dropdown in the header). That way, when analytics is trying to run the SQL query for these events, they can use the same column name for both searches

eg

`SELECT OLD_ORG_ID FROM [Table] WHERE OLD_ORG_ID= . . . `

Instead of having to call it `CURRENT_ORG_ID` in one SQL query and `OLD_ORG_ID` in another.

Here are the key names we're already using for switch org in OrgDropdown.tsx:

```
  const switchOrg = (org: TypeAheadMenuItem) => {
    event(HeaderNavEvent.OrgSwitch, multiOrgTag, {
      oldOrgID: activeOrg.id,
      oldOrgName: activeOrg.name,
      newOrgID: org.id,
      newOrgName: org.name,
    })
    window.location.href = `${CLOUD_URL}/orgs/${org.id}`
  }
```